sdk/client/src/listeners/network-listener/utils/web-socket-listener.ts

```diff
@@ -0,0 +1,41 @@
+import { WebSocketEvent, WebSocketRequest } from '../utils/models'
+
+export type WebSocketRequestListenerCallback = (event: WebSocketRequest) => void
+
+export type WebSocketEventListenerCallback = (event: WebSocketEvent) => void
+
+export type HighlightWebSocketWindow = Window & {
+	/**
+	 * Callback for web socket open and close events that are displayed in the network requests
+	 */
+	_highlightWebSocketRequestCallback: WebSocketRequestListenerCallback
+	/**
+	 * Callback for web socket message and error events that are displayed under the websocket requests
+	 */
+	_highlightWebSocketEventCallback: WebSocketEventListenerCallback
+	WebSocket: any
+}
+
+declare var window: HighlightWebSocketWindow
+
+const WebSocketListener = (
```

Comment:
Should we also block urls from the urlBlocklist here?

Follow up - what does blocking look like? Do we just not show the message, but that an event happen, or hide the event entirely?